.gitignore

```diff
@@ -30,3 +30,7 @@ include/
 arthur_bench/server/js/dist/*
 arthur_bench/server/js/node_modules/*
 arthur_bench/server/js/packages/*/node_modules/
+
+bench_runs
```

Comment:
Not 100% sure on including this, but I figured this is useful as my understanding of the `bench_runs` directory is that it is generated when running the server.

Reply:
yep this makes sense. the only files we would wanna track are the examples at `/examples/bench_runs`